src/matUtils/main.cpp

```diff
@@ -46,6 +47,8 @@ int main (int argc, char** argv) {
         mask_main(parsed); 
     } else if (cmd == "uncertainty") {
         uncertainty_main(parsed);
+    } else if (cmd == "merge"){

```

Comment:
@dtorvi you may want to add description of the merge function in the help message (line 21 onwards).